lib/server/chat-messages.service.ts

```
@@ -1,29 +1,35 @@
+import getContext from '../getContext';
 import { AI_MODEL } from '../consts';
+import getZoraPfpLink from '../zora/getZoraPfpLink';
+import { tool } from 'ai';
+
 export function createChatMessagesService() {
   return new ChatMessagesService();
 }
 
 class ChatMessagesService {
-  constructor() {}
+  constructor() { }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)